package.json

```diff
@@ -15,6 +15,7 @@
     "chart.js": "^2.9.4",
     "clone-deep": "^4.0.1",
     "debounce": "^1.2.0",
+    "dompurify": "^3.0.8",
```

Comment:
The creator is a respected hacker, so it should be covering a bunch of weird escapes.